### PR TITLE
Add split_info field in object.GetResponse body

### PR DESCRIPTION
**Pull request by alexvanin**
SplitInfo structure from split_info field contains meta information to assembly the object that has been split into parts.

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]

File: object/types.proto [hunk 1]
```diff
@@ -160,3 +160,21 @@ message Object {
   // Payload bytes.
   bytes payload = 4 [json_name = "payload"];
 }
+
+// Meta information of split hierarchy for object assembly. With right child
```
Comment (realloc) [inline object/types.proto:164]: Suggest to use "last object in the split list or something similar.

**Review by realloc — APPROVED** [review]

File: object/types.proto [hunk 2]
```diff
@@ -160,3 +160,21 @@ message Object {
...
+// one can traverse linked list back to left or first child to assembly
+// original object. With link object one can assembly object straight
+// away from the object parts.
+message SplitInfo {
+  // 16 byte UUID used to identify the split object hierarchy parts.
+  bytes split_id = 1;
+
+  // Identifier of most right child in split hierarchy parts. It contains
+  // split header with original object header.
+  neo.fs.v2.refs.ObjectID right_child_id = 2;
```
Comment (realloc) [inline object/types.proto:174]: Suggest to use 'last_part' or something similar

File: object/types.proto [hunk 3]
```diff
@@ -160,3 +160,21 @@ message Object {
...
+
+  // Identifier of linking object for split hierarchy parts. It contains
+  // split header with original object header and sorted list of
+  // object parts.
+  neo.fs.v2.refs.ObjectID link_id = 3;
```
Comment (realloc) [inline object/types.proto:179]: Maybe just 'link' here? It's ob ObejctID type, mabe there is no need to add `_id` suffix?

File: object/types.proto (hunk 1 above)
Comment (alexvanin) [reply object/types.proto]: Fixed.

File: object/types.proto (hunk 2 above)
Comment (alexvanin) [reply object/types.proto]: Fixed.

File: object/types.proto (hunk 3 above)
Comment (alexvanin) [reply object/types.proto]: Fixed.

File: object/types.proto [hunk 4]
```diff
@@ -160,3 +160,21 @@ message Object {
   // Payload bytes.
   bytes payload = 4 [json_name = "payload"];
 }
+
+// Meta information of split hierarchy for object assembly. With last part
+// one can traverse linked list of split hierarchy back to first part and
+// assembly original object. With linking object one can assembly object
```
Comment (realloc) [inline object/types.proto:166]: assemble the original object

**Review by realloc — APPROVED** [review]

File: object/types.proto (hunk 4 above)
Comment (alexvanin) [reply object/types.proto]: Fixed.

**Pull request merged**